modules/auxiliary/admin/networking/cisco_asa_extrabacon.rb

```diff
@@ -0,0 +1,229 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::SNMPClient
+  include Msf::Auxiliary::Cisco
+  include Msf::Exploit::Deprecated
+  moved_from 'auxiliary/admin/cisco/cisco_asa_extrabacon'
```

Comment:
Why is this now depreciated when it was released in 2016? Seems this would still be relevant? Or am I missing something here?

Reply:
This has now been resolved as this was a misunderstanding on my part. In reality this won't mark the module as depreciated unless the `depreciated` function is called. See discussion below.